docs/src/dev/developer/development-environment.asciidoc

```diff
@@ -497,6 +503,41 @@ docker/gremlin-server.sh 3.4.2
 To be a bit more clear, the version can not be a Docker tag like "latest" because there is no such TinkerPop artifact
 that has been published with that version number.
 
+[[docker-testing]]
+== Testing with Docker
+
+Some GLV allow you to run tests without Gremlin Server build. Currently supported gremlin-go, gremlin-javascript and
```

Comment:
I'm not sure what this first sentence is trying to explain.

Reply:
changed in https://github.com/Bit-Quill/tinkerpop/pull/25/commits/b58ec43eb69b2bcc41330a60c92f62cd74b3f5ed